test/test_premailer.rb

```diff
@@ -285,6 +285,22 @@ def test_premailer_related_attributes
     end
   end
 
+  def test_rgb
+    html = <<-END_HTML
+    <html> <head> <style>table { background-color: rgb(250, 250, 250); } </style>
+    <body>
+    <table> <tr> <td> Test </td> </tr> </table>
+    </body> </html>
+    END_HTML
+
+    [:nokogiri].each do |adapter|
+      pm = Premailer.new(html, :with_html_string => true, :rgb_to_hex => true, :adapter => adapter)
+      pm.to_inline_css
+      doc = pm.processed_doc
+      assert_equal 'FAFAFA', doc.at('table')['bgcolor']
+    end
```

Comment:
Add another style rule with a hex color to the example so we check that existing hex values don't get corrupted.